Gemfile.lock

```diff
@@ -442,7 +439,7 @@ DEPENDENCIES
   redis (~> 3.0)
   redis-rails
   resque
-  resque-retry
+  resque-scheduler
```

Comment:
Allows ActiveJob to use `enqueue_at` so we can schedule jobs for a specific time. (See <https://github.com/rails/rails/blob/6a8e7fd8d8cbfa335899153d455082cde906a579/activejob/lib/active_job/queue_adapters/resque_adapter.rb#L34>.)